content/millennium/r4/scheduling/slot.md

```diff
@@ -193,6 +193,63 @@ List an individual Slot by its id:
 
 The common [errors] and [OperationOutcomes] may be returned.
 
+## Patch
```

Comment:
We are supposed to have a beta tag for now, right?

Reply:
[done](https://github.com/cerner/fhir.cerner.com/pull/608/commits/28de14e4c8417bc7009f3aea3a1ede95607d30b6#diff-3251e7cb3d59e4aa73da18f3e0aa8a09553c8abaf90868d2811ccfced1e09d0bR198)

Reply:
+1 keeping this as a placeholder until we decide if a known issue tag needs to be added in this tag.

Reply:
updated the beta tag with known issues [here](https://github.com/cerner/fhir.cerner.com/pull/608/commits/293de4a74bdac6b27c4ef8f6059e60d6d4255df8)